src/moin/cli/maint/modify_item.py

```diff
@@ -144,8 +145,10 @@ def LoadHelp(namespace, path_to_help):
     Load an entire help namespace from distribution source.
     """
     logging.info("Load help started")
-    abspath_to_here = os.path.dirname(os.path.abspath(__file__))
-    path_to_items = os.path.normpath(os.path.join(abspath_to_here, path_to_help, namespace))
+    if path_to_help is None:
+        abspath_to_here = os.path.dirname(os.path.abspath(__file__))
+        path_to_help = os.path.abspath(os.path.join(abspath_to_here, '../../help/'))
+    path_to_items = os.path.normpath(os.path.join(path_to_help, namespace))
```

Comment:
making this work same as dump-help

Reply:
Perhaps you can try the same logic using moin_help as in 
https://github.com/moinwiki/moin/blob/bbd6ce2e9526ac8852b2b7f89d97d7c7c82e3225/src/moin/cli/maint/modify_item.py#L216
For me, it's easier to understand. What do you think about it?

Reply:
@UlrichB22 done, see https://github.com/moinwiki/moin/pull/1418/commits/51ce36dd344f43509028fb51837c92cdbe2d9e9b